analysis/benchmarking-functions/R/make_sce_list.R

```diff
@@ -0,0 +1,24 @@
+addCellQC <- function(sce, mito, threshold = 0){
```

Comment:
One other comment on this function, which came up when I was looking at https://github.com/AlexsLemonade/alsf-scpca/pull/101/files#r668161413 and I would think should be addressed especially if we add it to `scpcaTools`

The function name here follows the convention of `scater`, but I think that is potentially confusing in this case, because it does not match the other function names. We should follow the same naming convention for all of our functions. 

I also would prefer if the function name gave more of a hint of how it differs from `scater::addPerCellQC`. I would suggest a name like `add_mito_cell_qc` to indicate that it specifies the mitochondrial genes as a subset.